.github/workflows/main.yml

```diff
@@ -195,8 +196,10 @@ jobs:
     - name: "Prerequisties: Qt"
       shell: bash
       run: |
-        brew install qt5
-        ls -lha /usr/local/opt/qt/lib/cmake/Qt5
+        brew install qt@5
+        export LDFLAGS="-L/usr/local/Cellar/qt@5/5.15.2/lib"
+        export CPPFLAGS="-I/usr/local/Cellar/qt@5/5.15.2/include"
+        ls -lha /usr/local/Cellar/qt@5/5.15.2/lib/cmake/Qt5
```

Comment:
I'm unsure what this change does, qt5 should point at Qt5, not Qt6. This is even listed on the homebrew site itself as an alias to qt@5: https://formulae.brew.sh/formula/qt@5

Reply:
check the paths, qt5 breaks, qt@5 doesn't